G1ANT.Addon.Selenium/Api/SeleniumWrapper.cs

```diff
@@ -313,6 +313,39 @@ public void AlertPerformAction(string action, TimeSpan timeout)
             }
         }
 
+        public void Select(string selectValue, string selectBy, SeleniumCommandArguments search, TimeSpan timeout)
+        {
+            var element = GetElementInFrame(search, timeout);
+            var select = new SelectElement(element);
+
+            if (selectBy.ToLower()=="index")
+            {
+                try
+                {
+                    int index = int.Parse(selectValue);
+                }
+                catch (Exception)
+                {
+
+                    throw new ApplicationException($"{selectValue} is not an integer number");
+                }
+                select.SelectByIndex(int.Parse(selectValue));

```

Comment:
move this line into the try block

Reply:
I did it in another way  due to information of parsing error